src/ui/DNA/pages/MintDnaFlow/MintDnaFlow.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { Route, Routes } from 'react-router-dom';
-import { PageLayout } from 'src/ui/Onboarding/shared/PageLayout/PageLayout';
+import { PageLayout } from 'src/ui/components/PageLayout/PageLayout';
```

Comment:
Same here, this "PageLayout" is onboarding specific. Let's either rename or put in `src/ui/Onboarding/components`

Reply:
Moved to `src/ui/features/onboarding/shared`✅